packages/components/src/Menu/Menu.tsx

```diff
@@ -12,12 +11,16 @@ import classnames from "classnames";
 import { AnimatePresence, motion } from "framer-motion";
 import { useOnKeyDown } from "@jobber/hooks/useOnKeyDown";
 import { useRefocusOnActivator } from "@jobber/hooks/useRefocusOnActivator";
+import { useWindowDimensions } from "@jobber/hooks/useWindowDimensions";
 import { IconNames } from "@jobber/design";
-import { useSafeLayoutEffect } from "@jobber/hooks/useSafeLayoutEffect";
+import { usePopper } from "react-popper";
+import { useIsMounted } from "@jobber/hooks/useIsMounted";
+import ReactDOM from "react-dom";
 import styles from "./Menu.css";
 import { Button } from "../Button";
 import { Typography } from "../Typography";
 import { Icon } from "../Icon";
+import { focusAttribute } from "../FormField/hooks/useFormFieldFocus";
```

Comment:
this import location feels awkward.... pulling something from form field into `Menu` 

the alternative is moving it up to somewhere more generic and less tied to a specific component like `hooks`. I don't know if I like that either though because it is directly, and currently only used in the form field because that's currently the only place that has this problem